pretest.js

```diff
@@ -0,0 +1,28 @@
+if (!process.env.TEST_MYSQL_USER) {
+  console.log('not seeding DB with example db');
+  return;
+}
+
+var fs = require('fs');
+var cp = require('child_process');
+
+var sql = fs.createReadStream(require.resolve('./example/table.sql'));
+var stdio = ['pipe', process.stdout, process.stderr];
+var args = ['--user=' + process.env.TEST_MYSQL_USER];
+
+if (process.env.TEST_MYSQL_HOST) {
+  args.push('--host=' + process.env.TEST_MYSQL_HOST);
+}
+if (process.env.TEST_MYSQL_PORT) {
+  args.push('--port=' + process.env.TEST_MYSQL_PORT);
+}
+if (process.env.TEST_MYSQL_PASSWORD) {
+  args.push('--password=' + process.env.TEST_MYSQL_PASSWORD);
+}
+
+console.log('seeding DB with example db...');
+var mysql = cp.spawn('mysql', args, {stdio: stdio});
+sql.pipe(mysql.stdin);
+mysql.on('exit', function() {
+  console.log('done seeding DB');
```

Comment:
@rmg How are you handling errors? I see that mysql's stderr is forwarded to stderr, so at least we will see any error messages in console logs.

Should you check for the exit code of the `mysql` process and print a different error message & return non-zero exit code if the `mysql` process failed?